examples/flash_attention/example_gqa_bwd_tma_reduce_varlen.py

```
@@ -0,0 +1,792 @@
+        ctx.causal = causal
+        ctx.use_atomic = use_atomic
+        ctx.max_seqlen_q = max_seqlen_q
+        ctx.max_seqlen_k = max_seqlen_k
+        ctx.indices_q = indices_q
+        ctx.indices_k = indices_k
+        return o
+
+    @staticmethod
+    def backward(ctx, do):
+        N_CTX = do.shape[1]
+        q, k, v, o, lse, seqlens_q, seqlens_k, cu_seqlens_q, cu_seqlens_k = ctx.saved_tensors
```

Ruff (0.14.0)

Unpacked variable `seqlens_k` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)